tasks/kubectl-run-performance-tests.yml

```diff
@@ -27,6 +27,32 @@ run:
       gcloud auth activate-service-account --key-file ~/gcloud-service-key.json
       gcloud container clusters get-credentials ${KUBERNETES_CLUSTER} --zone europe-west2 --project ${GCP_PROJECT_NAME}
 
+      # THIS IS THE NEW PUBSUB PERFORMANCE TEST FANDANGO
+      kubectl scale deployment case-processor --replicas=0
```

Comment:
is this the right place to be scale apps? shouldn't the env already be in a state to be tested before this is run?

Reply:
oh actually ignore this comment, the penny has just dropped 👍

Reply:
We do this deliberately, so that we can count the number of receipt messages arriving on the `Case.Responses` queue.